WearStandaloneGoogleSignIn/app/build.gradle

```diff
@@ -23,13 +21,8 @@ android {
 }
 
 dependencies {
-    implementation 'com.google.android.gms:play-services-wearable:17.0.0'
```

Comment:
I'm guessing we aren't using any capabilityApis, DataApis, ChannelApis, or NodeApis in here. You can ignore this comment unless I add a followup saying I found one. Thanks!

Reply:
Ok, we don't, you can ignore.